.azure-pipelines/generation-pipeline.yml

```diff
@@ -330,7 +330,7 @@ stages:
         baseBranchName : 'feature/5.0'
         branchName: 'kiota/$(betaBranch)'
         targetClassName: "BaseGraphServiceClient"
-        targetNamespace: "MicrosoftGraphSdk"
+        targetNamespace: "Microsoft.Graph"
```

Comment:
actually sorry, I missed that in the initial PR. Don't you want to have beta somewhere in this one to enable side by side use of v1 and beta?

Reply:
Thanks for that. Created #719 to resolve.